src/__tests__/operations/property-filter.test.ts

```diff
@@ -101,17 +101,35 @@ describe('free text tokens', () => {
 
     expect(processed).toEqual([items[2], items[3]]);
   });
-  test('do not filter by a property, which does not support the operator of the token', () => {
```

Comment:
The change looks good overall but I would also keep the original test to verify the free-text filtering returns false (and true for negated) when matching properties that do not have the corresponding operator. For example, if using the `:` operator but the property only defines `=`, `!=`, the property must not be matched.

Reply:
Oh sorry, I meant to comment on the PR for this and forgot. [The current code today has this test duplicated](https://github.com/cloudscape-design/collection-hooks/blob/main/src/__tests__/operations/property-filter.test.ts#L92-L115), down to every character. Looks like a forgotten copy-paste.

So yes, the test you're asking for is still here, lines 92 to 103. 😅